src/test/java/it/gov/pagopa/payment/dto/mapper/TransactionInProgress2TransactionResponseMapperTest.java

```diff
@@ -20,25 +20,49 @@ void setUp() {
 
   @Test
   void applyTest() {
-    TransactionInProgress trx = TransactionInProgressFaker.mockInstance(1, SyncTrxStatus.CREATED);
+    TransactionInProgress trx = TransactionInProgressFaker.mockInstanceBuilder(1, SyncTrxStatus.CREATED)
```

Comment:
I'm torn between saying that mockInstanceBuilder is an utility method, and thus it should not be used, and sayng that returning the builder and setting further attributes directly in the test before the build could have prevented us to add the status parameter which I don't like that much 😃 
A middle ground could be using the mockInstance method and setting the specific attributes with the setter methods

Reply:
this is something introduced in previouses PR

Because the change on state was something really used, it was added as second parameter (it should be created an overloaded method without the status in order to let the previous default beahavior that was setting always CREATED)
But in order to close ASAP that PR, I approved that change...